frontend/src/views/Market.vue

```diff
@@ -531,6 +591,28 @@ export default Vue.extend({
         +this.activeType+' '+results.nftID+' to '+this.convertWeiToSkill(results.newPrice)+' SKILL';
     },
 
+    async updateNftListingTargetBuyer() {
+
+      this.marketOutcome = null;
+      if(this.selectedNftId === null) return;
+
+      let targetBuyer = await (this as any).$dialog.prompt({ title: `Sell ${this.activeType}`, text: 'Target Buyer Address (optional)' });
+      if(!targetBuyer) targetBuyer = defaultTargetBuyer;
+
+      this.waitingMarketOutcome = true;
+
+      const results = await this.changeMarketListingTargetBuyer({
+        nftContractAddr: this.contractAddress,
+        tokenId: this.selectedNftId,
+        newTargetBuyer: targetBuyer
+      });
+
+      this.selectedNftId = null;
+      this.waitingMarketOutcome = false;
+      this.marketOutcome = 'Successfully changed target buyer for '
```

Comment:
you could use template strings for this (`) to make it cleaner, like you did above

Reply:
Sure